SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -184,15 +169,41 @@ <h3>Be the first to answer this question!</h3>
 
     <!-- Display the answering form for logged in users -->
     {% if user.is_authenticated %}
-        <form id="answer_form" method="post">
-            {% csrf_token %}
-            <div id="wmd-button-bar"></div>
-            <textarea id="wmd-input" name="content" class="wmd-input"></textarea>
-            <div id="wmd-preview" class="wmd-panel wmd-preview"></div>
-            <input type="submit" name="answer_form" value="Submit your answer"/>
-        </form>
+        <button type="button" id="postA" data-toggle="modal" data-target="#formModal">Got Answers!? Click here to share them!</button>
```

Comment:
I did that since:
1. It makes more sense when it comes to maintaining consistency
2. The rigidity of the markdown library would have made keeping the answers textbox and having a modal for comments (i.e. another textbox) an invalid plan.